### ISSUE TITLE
Getting error while creating ionic project - serious issue

### ISSUE THREAD

**Issue by parveshhc** [ionic-team/ionic#11929]
✔ Creating directory ./release-hub - done!
[INFO] Fetching app base (https://github.com/ionic-team/ionic2-app-base/archive/master.tar.gz)
✔ Downloading - done!
[INFO] Fetching starter template blank (https://github.com/ionic-team/ionic2-starter-blank/archive/master.tar.gz)
✔ Downloading - done!
✔ Updating package.json with app details - done!
✔ Creating configuration file ionic.config.json - done!
[INFO] Installing dependencies may take several minutes!
> npm install
✖ Running command - failed!
[ERROR] An error occurred while running npm install (exit code 1):
        
        npm WARN deprecated object-keys@0.2.0: Please update to the latest object-keys
        npm ERR! Cannot read property 'find' of undefined
        npm ERR! Cannot read property 'find' of undefined
        
        npm ERR! A complete log of this run can be found in:
        npm ERR!     /Users/parvgarg/.npm/_logs/2017-06-05T16_57_08_280Z-debug.log

**Comment by danielsogl** [comment]
@parveshhc please fill out the error template. We can't help you without all required pieces of information.

**Comment by jgw96** [comment]
Hello! While we appreciate you opening an issue with us i am going to close this issue and ask that you open a new one that has the issue template filled out please. Thanks for using Ionic!

**Comment by parveshhc** [comment]
**Ionic version:**  (check one with "x")
[ ] **1.x** (For Ionic 1.x issues, please use https://github.com/ionic-team/ionic-v1)
[ ] **2.x**
[x] **3.x**

**I'm submitting a ...**  (check one with "x")
[x] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or http://ionicworldwide.herokuapp.com/

**Current behavior:**

ionic start release blank

**Expected behavior:**

Should create new blank project.

**Steps to reproduce:**


**Related code:**

```
insert any relevant code here
```

**Other information:**

> npm install
✖ Running command - failed!
[ERROR] An error occurred while running npm install (exit code 1):
        
        npm WARN deprecated object-keys@0.2.0: Please update to the latest object-keys
        npm ERR! Cannot read property 'find' of undefined
        npm ERR! Cannot read property 'find' of undefined
        
        npm ERR! A complete log of this run can be found in:
        npm ERR!     /Users/parvgarg/.npm/_logs/2017-06-05T17_09_03_157Z-debug.log
**Ionic info:** (run `ionic info` from a terminal/cmd prompt and paste output below):

```
insert the output from ionic info here
```

**Comment by geykel** [comment]
I had the same issue on OSX, fixed it by updating npm, had npm version 5.0.0 now 5.0.4.

Hope this helps.
